### ISSUE TITLE
docker support?

### ISSUE THREAD

**Issue by aniel300** [dtankdempse/free-iptv-channels#4]
any plans on supporting docker?

**Comment by dtankdempse** [comment]
I should have one setup in a day or two.

**Comment by aniel300** [comment]
that is good to hear, please let me know when it is available, thank you.

**Comment by dtankdempse** [comment]
It was added today.

**Comment by aniel300** [comment]
ok let me give it a try, thanks.

**Comment by aniel300** [comment]
i didn't know this project sync from "i.mjh.nz". is there a way to make it sync directly from the sources?

**Comment by dtankdempse** [comment]
Yes, as mentioned in the README section of the repository: 'It is based on the original script created by matthuisman...' If I decide to do it, it won't be any time soon.

**Comment by aniel300** [comment]
ok sounds good, thanks for everything.

**Comment by aniel300** [comment]
do u have a docker compose?

**Comment by dtankdempse** [comment]
I don't have one made yet, but I can look into creating one.

**Comment by aniel300** [comment]
ok thanks. please let me know here

**Comment by dtankdempse** [comment]
I added one to the Read Me section.

**Comment by philboycn** [comment]
looking forward to a linux/arm/v8 version.
